sdk/core/azure-core/azure/core/tracing/decorator.py

```diff
@@ -31,15 +31,23 @@
 from azure.core.settings import settings
 from azure.core.tracing.context import tracing_context
 
+try:
+    from typing import TYPE_CHECKING
+except ImportError:
+    TYPE_CHECKING = False
+
+if TYPE_CHECKING:
+    from typing import Callable, Any
+
 
 def distributed_trace(func=None, name_of_span=None):
-    # type: (Callable[[Any], Any], str) -> Callable[[Any], Any]
+    # type: (Callable, str) -> Callable[[Any], Any]
     if func is None:
         return functools.partial(distributed_trace, name_of_span=name_of_span)
 
     @functools.wraps(func)
-    def wrapper_use_tracer(self, *args, **kwargs):
```

Comment:
Oh, good. I noticed the other day and forgot to mention it. `*args, **kwarg`` is almost always the right signature for the inner wrapper